### PR TITLE
Update Istio sidecar annotation information for the Katib experiments

### PR DESCRIPTION
**Pull request by andreyvelich**
Fixes: https://github.com/kubeflow/katib/issues/1359.
Blocked by: https://github.com/kubeflow/website/pull/2312.

In the v1beta1 Katib version user should manually add `sidecar.istio.io/inject: "false"` to Experiment's Trial templates.

/assign @gaocegege @johnugeorge 
/cc @RFMVasconcelos @8bitmp3

### CODE REVIEW

File: content/en/docs/components/hyperparameter-tuning/experiment.md [hunk 1]
```diff
@@ -670,8 +670,18 @@ to launch an experiment from the command line:
...
-random algorithm example:
+**Note that**:
+
+- If you deployed Katib as part of Kubeflow, you have to change Kubeflow namespace to your profile namespace.
+
+- (Optional) Katib's experiments don't work with
+  [Istio sidecar injection](https://istio.io/latest/docs/setup/additional-setup/sidecar-injection/#automatic-sidecar-injection).
+  If you installed Kubeflow using [Istio config](https://www.kubeflow.org/docs/started/k8s/kfctl-k8s-istio/),
+  you have to disable sidecar injection. To do that, specify annotation `sidecar.istio.io/inject: "false"`
+  in experiment's trial template. See how to do it for `Job`, `TFJob` or `PyTorchJob` in
```
Comment (8bitmp3) [inline content/en/docs/components/hyperparameter-tuning/experiment.md:681]: Suggestion:

```suggestion
  in your experiment's trial template. For examples on how to do it for `Job`, `TFJob` (TensorFlow) or `PyTorchJob` (PyTorch), refer to the
```

File: content/en/docs/components/hyperparameter-tuning/experiment.md [hunk 2]
```diff
@@ -670,8 +670,18 @@ to launch an experiment from the command line:
 kubectl apply -f <your-path/your-experiment-config.yaml>
 ```
 
-For example, run the following command to launch an experiment using the
-random algorithm example:
+**Note that**:
```
Comment (8bitmp3) [inline content/en/docs/components/hyperparameter-tuning/experiment.md:673]: Suggestion:

```suggestion
**Note**:
```

File: content/en/docs/components/hyperparameter-tuning/experiment.md [hunk 3]
```diff
@@ -670,8 +670,18 @@ to launch an experiment from the command line:
...
+
+- If you deployed Katib as part of Kubeflow, you have to change Kubeflow namespace to your profile namespace.
```
Comment (8bitmp3) [inline content/en/docs/components/hyperparameter-tuning/experiment.md:675]:
```suggestion
- If you deploy Katib as part of Kubeflow, you have to change the Kubeflow namespace to your profile namespace.
```

File: content/en/docs/components/hyperparameter-tuning/experiment.md [hunk 4]
```diff
@@ -670,8 +670,18 @@ to launch an experiment from the command line:
...
+
+- (Optional) Katib's experiments don't work with
+  [Istio sidecar injection](https://istio.io/latest/docs/setup/additional-setup/sidecar-injection/#automatic-sidecar-injection).
+  If you installed Kubeflow using [Istio config](https://www.kubeflow.org/docs/started/k8s/kfctl-k8s-istio/),
```
Comment (8bitmp3) [inline content/en/docs/components/hyperparameter-tuning/experiment.md:679]:
```suggestion
  If you install Kubeflow using [Istio config](https://www.kubeflow.org/docs/started/k8s/kfctl-k8s-istio/),
```

File: content/en/docs/components/hyperparameter-tuning/experiment.md [hunk 5]
```diff
@@ -670,8 +670,18 @@ to launch an experiment from the command line:
...
+  you have to disable sidecar injection. To do that, specify annotation `sidecar.istio.io/inject: "false"`
+  in experiment's trial template. See how to do it for `Job`, `TFJob` or `PyTorchJob` in
+  [getting-started guide](http://localhost:1313/docs/components/hyperparameter-tuning/hyperparameter/#examples).
+
+Run the following command to launch an experiment using the random algorithm example:
 
 ```
```
Comment (8bitmp3) [inline content/en/docs/components/hyperparameter-tuning/experiment.md:686]: @andreyvelich If you have time, can you please add `shell` to three backticks whenever you see code blocks for command line examples? If you don't have time, don't worry. Thanks!

```suggestion
```shell
```

**Review by 8bitmp3 — COMMENTED** [review]
A few suggestions. Thank you @andreyvelich

File: content/en/docs/components/hyperparameter-tuning/experiment.md (hunk 5 above)
Comment (andreyvelich) [reply content/en/docs/components/hyperparameter-tuning/experiment.md:686]: > @andreyvelich If you have time, can you please add `shell` to three backticks whenever you see code blocks for command line examples? If you don't have time, don't worry. Thanks!

I'll do it, sure. Thanks!

**Review by andreyvelich — COMMENTED** [review]
@8bitmp3 Thank you for the review!
Your comments are addressed.

File: content/en/docs/components/hyperparameter-tuning/hyperparameter.md [hunk 6]
```diff
@@ -121,6 +121,18 @@ example:
...
+   in your experiment's trial template.
+
+   For the provided random example with Kubernetes [`Job`](https://kubernetes.io/docs/concepts/workloads/controllers/job/)
+   trial template, annotation should be under
+   [`.trialSpec.spec.template.metadata.annotations`](https://github.com/kubeflow/katib/blob/master/examples/v1beta1/random-example.yaml#L52).
+   For the Kubeflow `TFJob` or other training operators check
+   [here](https://www.kubeflow.org/docs/components/training/tftraining/#what-is-tfjob) how to set annotation.
+
 1. Deploy the example:
     ```
```
Comment (8bitmp3) [inline content/en/docs/components/hyperparameter-tuning/hyperparameter.md:137]: Thank you @andreyvelich One more thing - again, if you have time - can you add blank lines that separate code blocks and text paragraphs here and elsewhere (e.g. lines 327, 337)
```suggestion

    ```shell
```

Appreciate it.

/assign @gaocegege @johnugeorge @sperlingxx

(OWNERS/approvers other than @andreyvelich 👍)

**Comment by andreyvelich** [comment]
This PR is ready.
/cc @8bitmp3 @gaocegege @johnugeorge

**Review by gaocegege — COMMENTED** [review]
LGTM :+1:
/lgtm

**Pull request merged**